yahtzee.py

```diff
@@ -1,95 +1,60 @@
 #import packages
 import numpy as np
-import pygame as pg
-from win32api import GetSystemMetrics 
 import classes as c
 
-"""
-#set Display
-pg.init()
 
-display_width = GetSystemMetrics((0))
-display_height = GetSystemMetrics((1))
-
-#set colours
-black = (0, 0, 0)
-white = (255, 255, 255)
-
-#initialize the display
-gameDisplay = pg.display.set_mode((display_width,display_height))
-
-#instruction
-pg.display.set_caption('Yahtzee!')
-font = pg.font.Font('freesansbold.ttf', 8) 
-instruction = open('.\Instruction\Instruction_text.txt', 'r').read()
-text = font.render(instruction, True, black)
-textRect = text.get_rect()
-
-while True : 
-  
-    # completely fill the surface object 
-    # with white color 
-    gameDisplay.fill(white) 
-  
-    # copying the text surface object 
-    # to the display surface object  
-    # at the center coordinate. 
-    gameDisplay.blit(text, textRect)
+def game():
+    """
+    Function that can be used to play a game of Yahtzee. 
+    The game can be played with multiple players.
+    The scorecard of each player is updated in each turn.
+        
+    """
+
+    # ask with how many players you want to play the game, assigns integer to num_players
+    num_players = int(input('With how many players do you want to play Yahtzee?')) 
```

Comment:
it's a minor thing, but maybe make sure in the instruction that they type a number and not a word or alternatively add some code that can convert fifteen --> 15?